packages/address/src/types.ts

```diff
@@ -0,0 +1,48 @@
+export enum FieldName {
+  FirstName = 'firstName',
+  LastName = 'lastName',
+  Country = 'country',
+  City = 'city',
+  Zip = 'zip',
+  Province = 'province',
+  Address1 = 'address1',
+  Address2 = 'address2',
+  Phone = 'phone',
+  Company = 'company',
+}
+
+export interface Address {
+  company?: string;
+  firstName?: string;
+  lastName?: string;
+  address1: string;
+  address2: string;
+  city: string;
+  // Province code
+  province?: string;
+  zip: string;
+  // Country code of format: ISO 3166-1 alpha-2
```

Comment:
do we really care what the address format is? Maybe just specify this in the readme?

Reply:
True, that really needs to be in the README, but I thought it was cool to have it in the code. 😄

Reply:
I think I'd rather just call the variable `countryCode` or type is as an alias: `type CountryISOCode = string`